Neo4j.Driver/Neo4j.Driver/Internal/Connector/SocketConnection.cs

```diff
@@ -225,6 +216,12 @@ protected virtual void Dispose(bool isDisposing)
             }
         }
 
+        public Task CloseAsync()
+        {
+            Close();
+            return Task.CompletedTask; // TODO verify this is the correct way to do it
```

Comment:
@zhenlineo I thought to remove those Async methods on our second pass - as we really do not need them at all. Instead we should use Close().